frontend/components/TableContainer/DataTable/ProbabilityOfExploitCell.tsx/ProbabilityOfExploitCell.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+import { formatFloatAsPercentage } from "utilities/helpers";
+
+import Icon from "components/Icon";
+import { COLORS } from "styles/var/colors";
+
+import TextCell from "../TextCell";
+
+interface IProbabilityOfExploitCellProps<T> {
+  probabilityOfExploit?: number | null;
+  cisaKnownExploit?: boolean | null;
+  rowId: string;
+}
+
+const ProbabilityOfExploitCell = ({
```

Comment:
nice

Reply:
I realize that my work uses this icon outside of the context of a datatable. I'll make `ProbabilityOfExploit` component that can be wrapped in whatever cell-specific logic is needed when used in a table.